aws/ci_build

```diff
@@ -155,8 +155,13 @@ def main
         ChatClient.message 'server operations', commit_url, color: 'gray', message_format: 'text'
         DevelopersTopic.set_dtp 'yes'
         InfraProductionTopic.set_dtp_commit GitHub.sha('production')
-        zendesk_reminder = "/remind <@#{DevelopersTopic.dotd}> to check Zendesk in 2 hours"
-        ChatClient.message 'developers', zendesk_reminder
+        # You can find the channel id at the end of the url for a given channel using
+        # Slack in the browser, e.g https://codedotorg.slack.com/messages/<channel_id>
+        developers_channel_id = "C0T0PNTM3"
```

Comment:
Let's add this to configuration, e.g. `CDO.slack_developers_channel_id`, set in chef which will populate `globals.yml` on our deployment machines. It can be tested locally with `locals.yml`

Or if we want the option of storing multiple channel ids, something like this in the config:
```
slack_channel_ids: 
  developers: C0T0PNTM3
```
referenced in code via `CDO.slack_channel_ids['developers']`

Reply:
@wjordan is it okay with you if I put this info in the Chef secrets?